SwissTransferResources/Localizable/en.lproj/Localizable.strings

```diff
@@ -452,6 +458,18 @@
 /* loco:66c2f7b4302dbac2e60ec512 */
 "transferUploadSourceChoiceGallery" = "Photo and video gallery";
 
+/* loco:678e7b3b30ca99c9690c7c92 */
+"transferVirusCheckDescription" = "The files are not yet accessible for security reasons.\nPlease come back in a few moments.";
+
+/* loco:678e7b0a570febadc1013692 */
+"transferVirusCheckTitle" = "Files being checked";
+
+/* loco:678f4ab56e8b3c2390077452 */
+"transferVirusDetectedDescription" = "You can not access this transfer because it does not comply with our terms of use";
```

Comment:
Not sure if we should add the period. The other description has it.
```suggestion
"transferVirusDetectedDescription" = "You can not access this transfer because it does not comply with our terms of use.";
```